src/components/ImportFromCognitoS3Dialog/index.js

```diff
@@ -56,7 +34,7 @@ const customStyles = {
 }
 
 const ExpandedRow = ({ data }) => {
```

Comment:
i would move this to an adjacent file if possible to make the overall number of lines smaller.

Reply:
Which function/var are you refering to ?